Remove dollar signs from code blocks in local serve documentation page

KirstieJane opened the pull request:
<!--
Please complete the following sections when you submit your pull request. You are encouraged to keep this top level comment box updated as you develop and respond to reviews. Note that text within html comment tags will not be rendered.
-->
### Summary

<!-- Describe the problem you're trying to fix in this pull request. Please reference any related issue and use fixes/close to automatically close them, if pertinent. For example: "Fixes #58", or "Addresses (but does not close) #238". -->

Jupyter Book 2 does a wonderful job of highlighting code cells and making them easy to copy over to your terminal.

We had some lines in the page that started with `$ ` from before we had the JB2 functionality. When I copied and pasted those lines across the code didn't run as expected (because the `$` isn't actually supposed to be typed 😸 

### List of changes proposed in this PR (pull-request)

<!-- We suggest using bullets (indicated by * or -) and filled checkboxes [x] here -->

* Remove `$ ` from the start of lines in `community-handbook/local-build`

### What should a reviewer concentrate their feedback on?

<!-- This section is particularly useful if you have a pull request that is still in development. You can guide the reviews to focus on the parts that are ready for their comments. We suggest using bullets (indicated by * or -) and filled checkboxes [x] here -->

- [ ] Have I accidentally changed the meaning of any of our instructions?
- [ ] Everything looks ok?


### Acknowledging contributors

<!-- Please select the correct box -->

- [x] All contributors to this pull request are already named in the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file: <!-- replace this text with the GitHub IDs of any new contributors -->

JimMadge commented on book/website/reproducible-research/renv/renv-containers.md:
````diff
@@ -536,7 +536,7 @@ Create a new file in the `tmp` directory and close the container:
 
 Inspect the files in `tmp` on the host. The file `a.txt` was modified by the container process:
 ```console
-$ cat tmp/a.txt
+cat tmp/a.txt
 Hello
 World!
 ```
````
Would be worth leaving it here too as there is output.

KirstieJane replied:
Reverted - thank you!

https://github.com/the-turing-way/the-turing-way/blob/kw-jb2-formatting/book/website/reproducible-research/renv/renv-containers.md?plain=1#L539

JimMadge commented on book/website/ethical-research/activism/activism-env-impact.md:
Not sure what is happening with the changes here.

JimMadge reviewed (commented):
I think a lot of these are from me 😆, perhaps inspired by the Markdown linter I use.

When you have a code block showing a console command and output, like

```console
$ echo 'Hello world!'
Hello world!
```

it is good practice to include the prompt, because it makes it clear what is the command and what is the output.

Most of the cases you've found though don't have output, and the dollars prevent copying which is a pain.

da5nsy commented on book/website/reproducible-research/renv/renv-virtualmachine.md:
````diff
@@ -289,6 +289,6 @@ You can also destroy the virtual machine to clean up all storage associated with
 
 ```{code-block} console
 [vagrant@theturingway ~]$ exit
````
Was this one intentionally kept?

KirstieJane replied:
Hmmm - I remember seeing this one and not being sure what to do with it...

Do you have an opinion @da5nsy ? Is there a way that this looks the most clear?

da5nsy reviewed (dismissed):
Seconding @JimMadge's review

KirstieJane commented:
This is so great - thank you @JimMadge and @da5nsy for the review!

I think I've fixed the two that you've found with output now.... but it isn't showing up as updated in the PR for me 😩  I think the files DO show up as correct if you look at them in the branch.... so this feels like maybe GitHub is upset with me trying to change so many files at the same time?? Can you see if _you_ can see the two changes reverted back (the dollar signs should be there!)

I _think_ I've also reverted the environmental activism chapter - I bet that was a windows to linux conversion - thank you so much for catching!

One question that also came up for me when I was doing this is the difference between:

* ` ```{code-block} console `
* ` ```console `
* ` ```bash `

Do you know the difference? Is one of them more correct than another?

KirstieJane commented on book/website/community-handbook/infrastructure/infrastructure-dns.md:
````diff
@@ -47,7 +47,7 @@ dig [@DNS-server] [domain] [record-type]
 For example, we can check the A record for `book.the-turing-way.org` using Google's DNS server (`8.8.8.8`) like this
 
 ```console
-$ dig @8.8.8.8 book.the-turing-way.org A
+dig @8.8.8.8 book.the-turing-way.org A
 
 ; <<>> DiG 9.10.6 <<>> @8.8.8.8 book.the-turing-way.org A
 ; (1 server found)
````
I think I've reverted this back: https://github.com/the-turing-way/the-turing-way/blob/kw-jb2-formatting/book/website/community-handbook/infrastructure/infrastructure-dns.md?plain=1#L50

I'm not sure why GitHub isn't showing the change updated in this PR....
